src/pybind/mgr/cephadm/upgrade.py

```diff
@@ -25,6 +25,9 @@ def __init__(self,
                  target_version: Optional[str] = None,
                  error: Optional[str] = None,
                  paused: Optional[bool] = None,
+                 services_complete: Optional[List[str]] = None,
+                 progress_str: str = '',
```

Comment:
we should query this from the progress module.

Reply:
Would this just give us the % complete being used for the output of 'ceph -s' right now?

Reply:
this is already there!

https://github.com/ceph/ceph/blob/543fc009511219264c8890fa8b5b46fae2e8e77d/src/pybind/mgr/cephadm/upgrade.py#L261-L271